core/src/test/java/apoc/text/PhoneticTest.java

```diff
@@ -21,101 +21,11 @@ public void setUp() throws Exception {
-        testCall(db, "CALL apoc.text.phonetic('')", (row) ->
-                assertThat(row.get("value"), equalTo(""))
-        );
-        testCall(db, "RETURN apoc.text.phonetic('') as value", (row) ->
-                assertThat(row.get("value"), equalTo(""))
-        );
-    }
-
-    @Test
-    public void shouldComputeSoundexEncodingOfManyWords() {
-        testCall(db, "CALL apoc.text.phonetic('Hello, dear User!')", (row) ->
-                assertThat(row.get("value"), equalTo("H400D600U260"))
-        );
-        testCall(db, "RETURN apoc.text.phonetic('Hello, dear User!') as value", (row) ->
-                assertThat(row.get("value"), equalTo("H400D600U260"))
-        );
-    }
-
-
-    @Test
-    public void shouldComputeSoundexEncodingOfManyWordsEvenIfTheStringContainsSomeExtraChars() {
-        testCall(db, "CALL apoc.text.phonetic('  ,Hello,  dear User 5!')", (row) ->
-                assertThat(row.get("value"), equalTo("H400D600U260"))
-        );
-        testCall(db, "RETURN apoc.text.phonetic('  ,Hello,  dear User 5!') as value", (row) ->
-                assertThat(row.get("value"), equalTo("H400D600U260"))
-        );
-    }
-
     @Test
     public void shouldComputeSoundexDifference() {
         testCall(db, "CALL apoc.text.phoneticDelta('Hello Mr Rabbit', 'Hello Mr Ribbit')", (row) ->
                 assertThat(row.get("delta"), equalTo(4L))
         );
     }
 
-    @Test
-    public void shoudlComputeDoubleMetaphone() {
-        testCall(db, "CALL apoc.text.doubleMetaphone('Apoc')", (row) ->
-                assertThat(row.get("value"), equalTo("APK"))
-        );
-        testCall(db, "RETURN apoc.text.doubleMetaphone('Apoc') as value", (row) ->
-                assertThat(row.get("value"), equalTo("APK"))
-        );
-    }
-
-    @Test
-    public void shoudlComputeDoubleMetaphoneOfNull() {
-        testCall(db, "CALL apoc.text.doubleMetaphone(NULL)", (row) ->
-                assertThat(row.get("value"), equalTo(null))
-        );
-        testCall(db, "RETURN apoc.text.doubleMetaphone(NULL) as value", (row) ->
-                assertThat(row.get("value"), equalTo(null))
-        );
-    }
-
-    @Test
-    public void shoudlComputeDoubleMetaphoneForTheEmptyString() {
-        testCall(db, "CALL apoc.text.doubleMetaphone('')", (row) ->
-                assertThat(row.get("value"), equalTo(""))
-        );
-        testCall(db, "RETURN apoc.text.doubleMetaphone('') as value", (row) ->
-                assertThat(row.get("value"), equalTo(""))
-        );
-    }
-
-    @Test
-    public void shouldComputeDoubleMetaphoneOfManyWords() {
-        testCall(db, "CALL apoc.text.doubleMetaphone('Hello, dear User!')", (row) ->
-                assertThat(row.get("value"), equalTo("HLTRASR"))
-        );
-        testCall(db, "RETURN apoc.text.doubleMetaphone('Hello, dear User!') as value    ", (row) ->
-                assertThat(row.get("value"), equalTo("HLTRASR"))
-        );
-    }
```

Comment:
Same type of comment about the remaining test coverage of "apoc.text.doubleMetaphone"